lib/ble.toit

```diff
@@ -257,23 +270,157 @@ class Client:
     state := resource_state_.wait_for_state STARTED_EVENT_
     resource_state_.clear_state STARTED_EVENT_
 
+class ServerConfig:
+  resource_group_ := null
+  services/List := []
+
+  constructor:
+    resource_group_ = ble_server_config_init_
+
+  add_service uuid -> Service:
+    service := Service resource_group_ uuid
+    services.add service
+    return Service resource_group_ uuid
+
+
+/** 
+Defines a BLE service with characteristics 
+*/
+class Service:
+  /**
+  The UUID of the service.
+  
+  For 16 and 32 bit UUIDs, form the BLE variant with the top-level uuid function.
+  */
+  uuid/uuid_pkg.Uuid
+  
+  resource_group_/any
```

Comment:
The type of these should probably be `ByteArray?` because they are proxies for C++ objects.